vpplink/feature.go

```diff
@@ -17,12 +17,19 @@ package vpplink
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/projectcalico/vpp-dataplane/vpplink/binapi/vppapi/feature"
 	"github.com/projectcalico/vpp-dataplane/vpplink/binapi/vppapi/interface_types"
 )
 
+const (
```

Comment:
Shouldn't these constants rather go in the cnat file?

Reply:
Agreed !